### ISSUE TITLE
allow addition `rhgb quiet` to bootloader kernel args to be configurable

### ISSUE THREAD

**Issue by dustymabe** [rhinstaller/anaconda#1079]
Can we please make the addition of [rhgb quiet](https://github.com/rhinstaller/pykickstart/blob/b7176cc61b8e9f15e06e672af0d1805708584420/pykickstart/commands/bootloader.py#L96-L99) to the kernel arguments be configurable? It would be preferable to not have to [hack up the bootloader configuration using sed](https://bugzilla.redhat.com/show_bug.cgi?id=510523) in order to disable this functionality.

[Here](https://github.com/rhinstaller/anaconda/blob/ee91db6fa3972668b05808b3c7de125a7edd581f/pyanaconda/payload/__init__.py#L722-L726) is where this is handled today.

**Comment by dustymabe** [comment]
accidentally originally opened this issue against pykickstart: https://github.com/rhinstaller/pykickstart/issues/153. work would need to be done there too so I'll leave that issue open as well.

**Comment by jkonecny12** [comment]
This is a great idea. We have it in our future planning now. Thanks!

**Comment by rvykydal** [comment]
Agreed, I think we can't dare to change the present default behavior, so we could just add another option to bootloader command, something like --append-only= that would add only the value of the option without the stuff defined in anaconda as default (which is just rhgb and quiet currently)

**Comment by poncovka** [comment]
Hi, we add the boot options `rhgb quiet` only if `plymouth` is installed. You should be able to disable these options with:

```
%packages
-plymouth
%end
```

**Comment by dustymabe** [comment]
> Hi, we add the boot options `rhgb quiet` only if `plymouth` is installed

I see - I still would claim that it's worth making this configurable. How is one supposed to know (especially from the [docs](https://github.com/rhinstaller/anaconda/blob/ee91db6fa3972668b05808b3c7de125a7edd581f/pyanaconda/payload/__init__.py#L722-L726)) without going through the code to find it out? What if someone doesn't know what plymouth is but they know they don't want `rhgb quiet` on the kernel command line?

**Comment by poncovka** [comment]
Thank you for reporting this issue! The issue tracker will be closed on 3 June 2019. Please make sure that your issue is also reported at https://bugzilla.redhat.com/. We are handling a high number of issues and many of them involve other components. Bugzilla allows us to better process your issues and collaborate on them with other teams. We are sorry for the inconvenience.

**Comment by poncovka** [comment]
The kickstart documentation should be updated with: https://github.com/dcantrell/pykickstart/pull/270
